addons/event/models/event_registration.py

```diff
@@ -98,10 +98,10 @@ def default_get(self, fields):
                 ret_vals[field] = utm_mixin_defaults[mixin_field]
         return ret_vals
 
-    @api.depends('partner_id')
+    @api.depends('partner_id', 'partner_id.name')
```

Comment:
Already telling: we don't want that :) we just want an update when setting the partner_id but we don't want to synchronize. If you change the partner name, it should not impact registrations. It behaves like an onchange actually.

Reply:
@tde-banana-odoo yeah i know, i just temporarily pushed commit (+ wanted to see which tests to fail), because I don't fully get the functional part, I have to wait until luc comes back